Source/WebCore/page/TextIndicator.cpp

```diff
@@ -67,6 +69,20 @@ Ref<TextIndicator> TextIndicator::create(const TextIndicatorData& data)
     return adoptRef(*new TextIndicator(data));
 }
 
+RefPtr<TextIndicator> TextIndicator::createWithNode(Node& node, OptionSet<TextIndicatorOption> options,  TextIndicatorPresentationTransition presentationTransition, FloatSize margin)
+{
+    auto topMostSameLinkElement = &node;
```

Comment:
This variable should be renamed to be more generic, since the node is not necessarily a link (anchor) element.